WooCommerce/WooCommerceTests/ViewRelated/Orders/OrderListViewModelTests.swift

```diff
@@ -252,40 +252,17 @@ final class OrderListViewModelTests: XCTestCase {
                 break
             }
         }
-        viewModel.activate()
 
         // When
-        viewModel.updateBannerVisibility()
-
-        // Then
-        waitUntil {
-            viewModel.topBanner == .none
-        }
-    }
-
-    func test_when_having_no_error_and_orders_banner_should_be_shown_then_shows_orders_banner() {
-        // Given
-        let viewModel = OrderListViewModel(siteID: siteID, stores: stores, filters: nil)
-        stores.whenReceivingAction(ofType: AppSettingsAction.self) { action in
-            switch action {
-            case let .loadFeedbackVisibility(_, onCompletion):
-                onCompletion(.success(true))
-            default:
-                break
-            }
-        }
         viewModel.activate()
 
-        // When
-        viewModel.updateBannerVisibility()
-
         // Then
         waitUntil {
-            viewModel.topBanner == .orderCreation
+            viewModel.topBanner == .none
         }
     }
 
-    func test_when_having_no_error_and_orders_banner_visibility_loading_fails_shows_nothing() {
+    func test_when_having_no_error_and_banner_visibility_loading_fails_shows_nothing() {
```

Comment:
This test should be removed because we are no longer loading banner visibility.

Reply:
Good catch, removed on https://github.com/woocommerce/woocommerce-ios/pull/11582/commits/c2741f1d3c34c7366a7b96be6df5d77b8fd5914f